tailwind.config.js

```diff
@@ -22,6 +51,10 @@ module.exports = {
           //secondary
           'secondary-deep': '#3a6b90',
           'dull': '#5996b2',
+
```

Comment:
would it be possible to organize the colours in the same way as defined, so gray-light etc as these colours are used elsewhere. Just change in the jsx to match